plugins/com.google.cloud.tools.eclipse.sdk.test/pom.xml

```diff
@@ -1,10 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Okay, I'll keep `Bundle-ClassPath: .` and add it to all bundles that miss this.

I think I'll still skip `<sourceDirectory>src</sourceDirectory>`, since otherwise there will be too many, and FWIW, the Maven philosophy is to not specify things unless it's not default. The "src" directory is too obvious too.